src/app-layout/__integ__/runtime-drawers.test.ts

```diff
@@ -77,7 +77,7 @@ describe.each(['classic', 'refresh', 'refresh-toolbar'] as const)('%s', theme =>
 });
 
 describe('Visual refresh toolbar only', () => {
-  function setupTest(testFn: (page: BasePageObject) => Promise<void>) {
+  function setupTest(testFn: (page: BasePageObject, browser: WebdriverIO.Browser) => Promise<void>) {
```

Comment:
This typing is not cross-version compatible. You will make Andrei's life upgrading webdriverio version easier, if you use a more portable version

Reply:
Yes, this type is not even imported here in this file

Reply:
because this type is globally available: https://webdriver.io/docs/api/globals/

what exactly do you want me to change to make it forward compatible?

Reply:
Follow this pattern: https://github.com/search?q=repo%3Acloudscape-design%2Fcomponents%20browser%3A%20ConstructorParameters&type=code

Reply:
done